tests/conftest.py

```diff
@@ -79,3 +80,13 @@ def _import_execute(source: str, *, custom_module_name: str = None):
             print('KeyboardInterrupt')
 
     return _import_execute
+
+
+if sys.platform == 'emscripten':
+
+    @pytest.fixture
+    def benchmark():
+        def result(func, *args, **kwargs):
+            return func(*args, **kwargs)
+
+        return result
```

Comment:
Looks like we might need to add a marker here to avoid all the warnings:

```
../../test_dir/benchmarks/test_micro_benchmarks.py:60
  /test_dir/benchmarks/test_micro_benchmarks.py:60: PytestUnknownMarkWarning: Unknown pytest.mark.benchmark - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.benchmark(group='simple model - python')
```

This is working for me:

```py
def pytest_configure(config):
    config.addinivalue_line('markers', 'benchmark: pytest-benchmark stub, ignored')


def pytest_addoption(parser):
    parser.addoption('--benchmark-columns', action='store', default='-', help='pytest-benchmark stub, ignored')
    parser.addoption('--benchmark-group-by', action='store', default='-', help='pytest-benchmark stub, ignored')
    parser.addoption('--benchmark-warmup', action='store', default='-', help='pytest-benchmark stub, ignored')
    parser.addoption('--benchmark-disable', action='store_true', default='-', help='pytest-benchmark stub, ignored')
```

Reply:
👍